rust/src/container.rs

```diff
@@ -78,6 +78,10 @@ struct ContainerEncapsulateOpts {
     #[clap(long)]
     /// Output content metadata as JSON
     write_contentmeta_json: Option<Utf8PathBuf>,
+
+    /// Compare OCI layers of current build with another(imgref)
+    #[clap(name = "compare-with-build", long, short)]
```

Comment:
Did you intend to have `short` here too?  I think that will expose this as `-c` which is probably fine, but I tend not to use short options unless they're very commonly used, and it's not clear to me this one will be.